app/components/window/Titlebar.tsx

```diff
@@ -143,7 +154,7 @@ export const Titlebar = () => {
               onClick={() => {
                 if (
                   confirm(
-                    'Are you sure you want to install the update? The app will quit to install.',
+                    'Are you sure you want to install the update? The app will restart.',
```

Comment:
Reverting this text change as windows + mac auto updater now restarts the app. Opening the app as it installs the update will show some errors as electron-updater swaps out the files, but it will complete.